src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+  .calendar {
+    flex-wrap: wrap;
+    gap: $calendar-gap;
+    width: calc(7 * $calendar-day-width + 7px);
```

Comment:
The value `7px` seems arbitrary and does not follow a variable or calculation that relates to the gap or other dimensions. Ensure that all spacing follows the design variables for consistency and maintainability.